
<launch>

<node pkg="cartesian_move" type="relative_move_node" name="relative_move" output="screen">
    <param name="ee_frame" value="mur/ur/tool0"/>
    <param name="base_frame" value="mur/ur/base_link"/>
    <param name="planning_frame" value="mur/mir/base_link"/>
    <param name="rate" value="30"/>
</node>

</launch>